recipes/yt/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "3.2.3" %}
+
+package:
+  name: yt
+  version: {{ version }}
+
+source:
+  fn: yt-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/y/yt/yt-{{ version }}.tar.gz
+  sha256: 4d6ccf345d9fab965335c9faf8708c7eea79366b81d77f0f302808be3e82c0ed
+
+build:
+  entry_points:
```

Comment:
Should have the following.

    script: python setup.py --quiet install --single-version-externally-managed --record=record.txt

Then delete `build.sh`.